recipes/gwvet/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "gwvet" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "601846a8a201c6950d54f57628147bc87fc9b1f2b9a502616941a9cc5e531dcb"
+
+build:
+  number: 0
+  skip: true  # [win or py<36]
```

Comment:
This can be `noarch: python`. If windows builds fail because of missing dependencies on staged-recipes, this is ok. On the feedstock, we will build only on Linux. You can declare the python version constraint then in `host` and `run`.

Reply:
Thank you @xhochy! I've implemented these changes, new builds are running as we speak.

Reply:
@xhochy, windows builds have indeed failed because of missing dependencies, but linux and OSX builds were successful.